app/views/flights/show.json.jbuilder

```diff
@@ -4,8 +4,9 @@ json.flight do
     json.partial! 'contest', contest: @flight.contest
     json.url contest_url(@flight.contest, :format => :json)
   end
-  json.category do
-    json.partial! 'categories/category', category: @flight.category
+  json.categories do
+    json.array! @flight.categories,
+      partial: 'categories/category', as: :category
```

Comment:
@djmolny this might be a breaking change for the judge experience processing in iac.org. Instead of {"category" =>{blah blah}}` it will produce `"categories" => [{blah blah}]`. I'll investigate. I think the way to check is to find the code on the server? Maybe we consider putting those codes in a source code repository somewhere. Or maybe I alter this so that it continues to make the same output, only using the first of the categories.

Reply:
Never mind. This code goes to the REST API, which is something other than the judge/activity endpoint that the judge credit is using. Win.